code-challenges.js

```diff
@@ -7,43 +7,130 @@
+// Then console.log using .reverse() to make the array order go in reverse using the new variable alpha prior the mutator indicating as Alpha 2022.
+// Then conmsole.log using .join(" ") with a space to separate the characters and make a string by converting the values.
+//Therefore the following codes meet the conditions to output 2202 ahplA.
+
+
+// var alpha= currentCohort.split("")
+// console.log(alpha.reverse())
+// console.log(alpha.join(" "))
 
 
 // --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
 
-const myArray = [13, 34, 5, 10, 27, 42]
+// const myArray = [13, 34, 5, 10, 27, 42]
+
+// Pseudocode:
+// Create a statement that evaluates whether it's even or odd within the given numbers of the array.
+// Use a for loop declaration stating that arrays myArray is less than one to multiply.
+// Use the declaration const name num that equates to myArray with a value of i = 0 in a bracket to state that identifier doesn't change.
+// Use an If/Else statement to see if the program will continue to run when the set condition is true, plugging in the new const name nuum modulo 2 with a Loose Equality operatorn == and 0 all being in parenthese and this will compare the two values for equality.
+// Then console.log with parentheses sqying the number is even in quotation for even numbers.
+// Then console.log with parentheses saying the number is odd in quotation for odd numbers.
+// Therefore the output desired is displaying in the terminal because the conditions are met.
+
+
+// for(let i=0; i<myArray.length; i++){
+// const num = myArray[i]
+// if(num % 2 == 0) {
+//     console.log("even")
+// }
+// else
+//     console.log("odd")
+// }
 
 
 // --------------------5) Create a function that takes in a set of any two numbers and subtracts the smaller number from the larger number using the sets of test variables provided below. Expected output: 42, 3
 
 // Set one:
-const number1 = 58
-const number2 = 100
+// const number1 = 58
+// const number2 = 100
 
 // Set two:
-const number3 = 27
-const number4 = 24
+// const number3 = 27
+// const number4 = 24
+
+// Pseudocode:
+// Create a function that performs a mathematical equation that substracts the given test variables in smaller to larger number.
+// I need to create a function named num to equate a set of variable to take in two numbers.
+// The parameters will take in num 1 and num 2.
+// If num1 is less than num2 that means that num2 is greater than, which will then subtract the larger number num2 minus num1.
+// If num2 is less than num1 that means that num1 is less than, which will then subtract the larger number num1 minus num2.
+// I will return the difference between the larger and smaller number.
+// I can now make sure that both sets are able to communicate with the return.
+
+
+// var num = (num1, num2) => {
```

Comment:
It is a best practice to declare your functions with `const`

Reply:
sorry i was following the syntax in the syllabus & took it too literally -_-